apps/fm/js/fm.js

```diff
@@ -184,11 +184,24 @@ function updatePowerUI() {
 }
 
 function updateAntennaUI() {
-  $('antenna-warning').hidden = mozFMRadio.antennaAvailable;
```

Comment:
I think we can do it a little bit simpler. Instead of tracking the state (based on which warning to display etc). Lets just do the following:
in updateAntennaUI and updateAirplaneModeUI we should just add 'warning' class to the document body if the warning is displayed as follows:
```
document.body.classList.toggle('warning', !mozFMRadio.antennaAvailable || airplaneModeEnabled);
```
Then in fm.css just have this styling:
```
.warning #container {
  visibility: hidden;
}
```